torch/csrc/jit/script/schema_type_parser.cpp

```diff
@@ -21,9 +21,17 @@ TypeAndAlias SchemaTypeParser::parseBaseType() {
       {"float", FloatType::get()},
       {"int", IntType::get()},
       {"bool", BoolType::get()},
+      {"None", NoneType::get()},
   };
-  auto tok = L.expect(TK_IDENT);
-  auto text = tok.text();
+  auto tok = L.cur();
+  std::string text;
+  if (L.nextIf(TK_NONE)) {
+    text = "None";
```

Comment:
Why is this necessary? Seems like the text of TK_NONE is "None" in lexer.h

Reply:
Previously the lexer was expecting an ident but found TK_NONE, causing an error. Yes instead of `text = "None"` I could write  `L.cur().text()` if that's what you're asking